packages/openapi/src/api.ts

```diff
@@ -3371,6 +3371,13 @@ export interface components {
       | components["schemas"]["Banner"]
       | components["schemas"]["MissingPackageAlert"]
       | components["schemas"]["InstallingPackageAlert"]
+      | {
+          content: string;
+          /** @enum {string} */
+          name: "startup-logs";
+          /** @enum {string} */
+          status: "append" | "start" | "done";
+        }
```

Comment:
planning to wait for @manzt's `msgspec` to merge to avoid a merge conflict, then will update this